### PR TITLE
UB-119: Install pyscopg2 when running the tests

### PR DESCRIPTION
**Pull request by tzulberti-jampp**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+def parse_requirements_txt(filename='requirements.txt'):
+    with open(os.path.join(current_dir, filename)) as requirements_file:
+        requirements = requirements_file.readlines()
+        # remove whitespaces
+        requirements = [line.strip().replace(' ', '') for line in requirements]
+        # remove all the requirements that are comments
+        requirements = [line for line in requirements if not line.startswith('#')]
+        # remove empty lines
+        requirements = list(filter(None, requirements))
+        return requirements
```
Comment (danielfujii-jampp) [inline setup.py]: I don't think we've done this before. Why now?

Comment (tzulberti-jampp) [reply setup.py]: Yes, that is the new way to read the requirements files on the setup.py.

Comment (danielfujii-jampp) [reply setup.py]: Is there no other way? Like executing a command with sys?

Comment (tzulberti-jampp) [reply setup.py]: I don't understand you comment. The function just parses the file, it doesn't install any dependency

Comment (danielfujii-jampp) [reply setup.py]: Yes, I know, I just don'e like having to write code to do this. Seems like running a `pip install -r requirements.txt` is safer.

Comment (tzulberti-jampp) [reply setup.py]: That is why I created the requirements.txt as an external file. But when installing gcd as an external dependency you can't do that. That is why I am parsing the file on the setup.py

**Pull request merged**